doc/roadmap.rst

```diff
@@ -46,7 +46,7 @@ One Year Plan Through 2021
 Modularize HNN code to simplify installation, development and maintenance
 -------------------------------------------------------------------------
```

Comment:
```suggestion
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

See [RST cheat sheet](https://docs.anaconda.com/restructuredtext/index.html), I believe these should be 4th level headings (under "One Year Plan...")